src/Stryker.Core/Stryker.Core/Reporters/ClearTextReporter.cs

```diff
@@ -78,7 +78,13 @@ private void DisplayComponent(IReadOnlyProjectComponent inputComponent, Table ta
 
             var mutationScore = inputComponent.GetMutationScore();
 
-            if (inputComponent.IsComponentExcluded(_options.Mutate))
+            var isExcluded = inputComponent switch
```

Comment:
Trying to understand this... is this correct? Can a folder be excluded in the context of clear text reporters?